lab-matt/app/controller/game-controller.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const angular = require('angular');
+
+
+const Monster = require('../model/monster');
+
+
+angular.module('adventureApp').controller('GameController', [GameController]);
+
+function GameController(){
+  this.history = ['Welcome to Space Jail.  Your cell recently malfunctioned and open.  Try to escape!'];
+  this.moveCount = 0;
+  this.map = require('../model/map');
+
+  this.player = {
+    name: 'Space Ash',
+    spaceSuit: 100,
+    location: 'roomA',
+    damage: 10,
+    xp: 0
+  };
+
+  this.gameForm = {
+    direction: 'North'
+  };
```

Comment:
I've seen this a couple times now. Is this how Duncan wanted you to do it? It makes more sense to me to just have:
`this.direction = 'North';`
or even just declare it on the form's scope with ng-init, but whatevs